calculator/src/tokens.kt

```diff
@@ -4,10 +4,11 @@ sealed class Token
 data class Floating(val value: Float) : Token()
 data class Integer(val value: Int) : Token()
 
-//singleton
 object LeftParenthesis : Token()
 object RightParenthesis : Token()
 
+// TODO: Change the classes to objects
```

Comment:
perhaps adhere to the class structure presented in shunting branch or present a new standard for all branches to use

Reply:
I have a different branch which changes the class hierarchy of the tokens. It will be merged in the second sprint.